Gopkg.toml

```diff
@@ -514,28 +514,28 @@ required = [
 
 [[constraint]]
   name = "k8s.io/api"
-  revision = "kubernetes-1.15.0"
+  revision = "kubernetes-1.15.1"
 
   # main-usage = "pkg/k8s"
   # on-revision = ""
 
 [[constraint]]
   name = "k8s.io/apiextensions-apiserver"
-  revision = "kubernetes-1.15.0"
+  revision = "kubernetes-1.15.1"
```

Comment:
How come there are no changes to `vendor/` associated with this change? Are we not using any of the files in K8s that were changed in k8s 1.15.0 --> 1.15.1?

Reply:
Yes, it can happen.